docker/swift-t/Dockerfile

```diff
@@ -10,31 +10,45 @@ FROM 136a88385611
 ENV SWIFT_ROOT=/opt/swift-t
 
 # Install dependencies
-RUN yum -y install mpich-devel # OpenMPI is fine too
+# RUN yum -y install mpich-devel
+RUN yum -y install openmpi-devel
 RUN yum -y install swig
 RUN yum -y install zsh
```

Comment:
Unfortunately with Docker, each RUN results in a whole new layer in the dockerfile, and there are typically only 128 layers allowed (apparently it is actually an undocumented amount: docker/docker.github.io#8230).

It is gross, but the "best practice" is to chain all of these together into a single RUN to save layers.